services/backend/src/assignments-controller.js

```diff
@@ -34,7 +34,7 @@ async function getAssignment(req, res) {
 }
 
 async function getAssignments(req, res) {
-    const requestedBy = req.auth.uid;
+    const requestedBy = req.params.uid;
```

Comment:
This should still be `req.auth.uid`.

`req.params` refers to route parameters like `/assignments/:aid` and `req.auth` is created in the middleware function `handleAuth`. The reason you're not seeing it here is because you're calling the request handler directly (and therefore skipping the auth middleware). All you need to do is add an auth object to the request. You can see how I do it in my user tests.

Reply:
All the places you changed it should be changed back